.gitignore

```diff
@@ -24,3 +24,5 @@ spec/dummy/public/uploads/*
 .rubocop.yml
 
 *.gem
+
+vendor/
```

Comment:
since we're removing `vendors` from the repo and this helps with your local issue, we can leave it!